### PR TITLE
Added accessibility policy body component.

### PR DESCRIPTION
**Pull request by dsjen**
This adds a component for displaying the body of the accessibility request page. The phone number needs to be updated when it's settled.

@edx/educator-dahlia

### CODE REVIEW

**Comment by dsjen** [comment]
I just realized that I should probably wrap this in a bootstrap grid. Feel free to review still, and I'll update this in a bit.

File: src/components/AccessibilityBody/index.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
```
Comment (MichaelRoytman) [inline src/components/AccessibilityBody/index.jsx]: Should this be `mailTo`? Or is it clearer to leave it as mailto, since it's referring to the URI?

File: src/components/AccessibilityBody/index.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+
+  const communityAccessibilitySite = `${props.communityAccessibilityLink}`;
```
Comment (MichaelRoytman) [inline src/components/AccessibilityBody/index.jsx]: I see that `communityAccessibilityLink` is a String in the PropTypes. Can you explain the use of the template string? Does` const communityAccessibilitySite = props.communityAccessibilityLink;` not work?

File: src/components/AccessibilityBody/index.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  const communityAccessibilityElement = (
+    <a href={communityAccessibilitySite}>Website Accessibility Policy</a>
+  );
+
+  /* eslint-disable max-len */
```
Comment (MichaelRoytman) [inline src/components/AccessibilityBody/index.jsx]: Out of curiosity, why disable the max-len rule? Does breaking the long text into shorter lines affect how it's displayed in the browser?

File: src/components/AccessibilityPolicyPage/index.jsx [hunk 4]
```diff
@@ -3,6 +3,8 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 // import styles from './AccessibilityPolicyPage.scss';
+import AccessibilityBody from '../AccessibilityBody';
```
Comment (MichaelRoytman) [inline src/components/AccessibilityPolicyPage/index.jsx]: I just realized that the form is probably technically part of the "body", but I think that's probably super inconsequential.

File: src/components/AccessibilityBody/AccessibilityBody.test.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        <AccessibilityBody
+          communityAccessibilityLink="http://example.com"
+          phoneNumber="555-555-5555"
+          email="example@example.com"
+        />,
+      );
+    });
+    it('contains links', () => {
+      expect(wrapper.text()).toContain('example@example.com');
+      expect(wrapper.text()).toContain('555-555-5555');
```
Comment (MichaelRoytman) [inline src/components/AccessibilityBody/AccessibilityBody.test.jsx]: Is there a way to add an expect for the communityAccessibilityLink?

**Comment by dsjen** [comment]
FYI, I just pushed changes to wrap the page in a bootstrap container and also added the a11y and assets page urls to the readme.

File: src/components/AccessibilityBody/index.jsx (hunk 1 above)
Comment (dsjen) [reply src/components/AccessibilityBody/index.jsx]: Yeah, I was sort of debating that, and just went with the lowercase one as it's consistent with the attribute. I'm inclined to leave it as is.

File: src/components/AccessibilityBody/index.jsx (hunk 2 above)
Comment (dsjen) [reply src/components/AccessibilityBody/index.jsx]: Good catch! I was futzing around with something and can get rid of this. I'll update to:
```
  const communityAccessibilityElement = (
    <a href={props.communityAccessibilityLink}>Website Accessibility Policy</a>
  );
```

File: src/components/AccessibilityBody/index.jsx (hunk 3 above)
Comment (dsjen) [reply src/components/AccessibilityBody/index.jsx]: It doesn't affect the display, but I didn't want to have to muck around with formatting the text essentially. So...disabling the line count is partially out of laziness, but also in that we get the copy from legal and I'd rather just copy/paste.

File: src/components/AccessibilityBody/AccessibilityBody.test.jsx (hunk 5 above)
Comment (dsjen) [reply src/components/AccessibilityBody/AccessibilityBody.test.jsx]: Sure thing!

I'll add in something like:
```
      expect(wrapper.find({ href: 'http://example.com' })).toHaveLength(1);
```

**Comment by MichaelRoytman** [comment]
Thanks for adding documentation around accessing this page, too!

File: src/components/AccessibilityBody/index.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
+  const communityAccessibilityElement = (
+    <a href={props.communityAccessibilityLink}>Website Accessibility Policy</a>
+  );
+
+  /* eslint-disable max-len */
+  return (
```
Comment (jaebradley) [inline src/components/AccessibilityBody/index.jsx]: Does all this text need to be internationalized? Or is that not in scope or relevant?

Comment (MichaelRoytman) [reply src/components/AccessibilityBody/index.jsx]: I don't believe it needs to be internationalized. Only the link in the footer to this page needs to be.

File: src/components/AccessibilityBody/index.jsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+          <li>How soon you need access and for how long (e.g., a planned course start date or in connection with a course-related deadline such as a final essay).</li>
+        </ul>
+        <li>Within 48 business hours of receiving your request, the edX Support Team will respond to confirm receipt and forward your request to the edX Partner Manager for your institution and the edX Website Accessibility Coordinator.</li>
+        <li>With guidance from the Website Accessibility Coordinator, edX will contact you within 5 business days to discuss your request and gather additional information from you to identify a solution.</li>
+        <li>EdX will assist you promptly and thoroughly so that you are able to create content on the CMS within your time constraints.  Such efforts may include, but are not limited to: (a) purchasing a third-party tool or software for use on an individual basis to assist your use of Studio, (b) engaging a trained independent contractor to provide real-time visual, verbal and physical assistance, or (c) developing new code to implement a technical fix.</li>
```
Comment (jaebradley) [inline src/components/AccessibilityBody/index.jsx]: Super (Grammar) Nit: it's `edX` in some places and `EdX` in others - I think it should be `edX`?

File: src/components/AccessibilityBody/index.jsx (hunk 1 above)
Comment (jaebradley) [reply src/components/AccessibilityBody/index.jsx]: You could also just forgo creating the variable and just stick it in the `href` directly. It doesn't look like `mailto` is being used anywhere else.

File: src/components/AccessibilityBody/index.jsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccessibilityBody = (props) => {
+  const mailto = `mailto:${props.email}`;
+  const emailElement = (<a href={mailto}>{props.email}</a>);
```
Comment (jaebradley) [inline src/components/AccessibilityBody/index.jsx]: Just FYI, we now have a [`Hyperlink` `Paragon` component](https://github.com/edx/paragon/blob/master/src/Hyperlink/index.jsx)

**Comment by dsjen** [comment]
@jaebradley -- this PR is in regards to specific legal text.
